### PR TITLE
feat(deps): version bump for 5.2.1 release

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
Bump for our 5.2.1 patch release.

### CODE REVIEW

File: packages/documentation-framework/versions.json [hunk 1]
```diff
@@ -1,9 +1,33 @@
 {
   "Releases": [
+    {
+      "name": "5.2.1",
+      "date": "2024-02-05",
```
Comment (evwilkin) [inline packages/documentation-framework/versions.json:5]:
```suggestion
      "date": "2024-03-05",
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 2]
```diff
@@ -7,6 +7,32 @@ section: get-started
...
+  - [@patternfly/react-table@5.2.1](https://www.npmjs.com/package/@patternfly/react-table/v/5.2.1)
+  - [@patternfly/react-tokens@5.2.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/5.2.1)
+
+### Fixes include:
+- Issues with styles and intractability of cards in Chromium based browsers
+- Support for status modifiers on Modal titles
+- Type updates to react-charts to reflect changes in the Victory types
+- Reversion of a previous type change with our clipboard copy component that was unintentionally breaking
+- Pass through of more props to monaco-editor in our code editor component
+
```
Comment (evwilkin) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:35]:
```suggestion

<Divider />

```

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 3]
```diff
@@ -7,6 +7,32 @@ section: get-started
...
+  - [@patternfly/react-code-editor@5.2.1](https://www.npmjs.com/package/@patternfly/react-code-editor/v/5.2.1)
+  - [@patternfly/react-core@5.2.1](https://www.npmjs.com/package/@patternfly/react-core/v/5.2.1)
+  - [@patternfly/react-drag-drop@5.2.1](https://www.npmjs.com/package/@patternfly/react-drag-drop/v/5.2.1)
+  - [@patternfly/react-icons@5.2.1](https://www.npmjs.com/package/@patternfly/react-icons/v/5.2.1)
+  - [@patternfly/react-styles@5.2.1](https://www.npmjs.com/package/@patternfly/react-styles/v/5.2.1)
+  - [@patternfly/react-table@5.2.1](https://www.npmjs.com/package/@patternfly/react-table/v/5.2.1)
+  - [@patternfly/react-tokens@5.2.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/5.2.1)
+
+### Fixes include:
+- Issues with styles and intractability of cards in Chromium based browsers
```
Comment (evwilkin) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:30]:
```suggestion
- Issues with styles and inetractability of cards in Chromium based browsers
  - Additional details: https://github.com/orgs/patternfly/discussions/6381
```

File: packages/documentation-framework/versions.json [hunk 4]
```diff
@@ -1,9 +1,33 @@
 {
   "Releases": [
+    {
+      "name": "5.2.1",
+      "date": "2024-03-05",
```
Comment (wise-king-sullyman) [inline packages/documentation-framework/versions.json:5]:
```suggestion
      "date": "2024-03-01",
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 5]
```diff
@@ -7,6 +7,35 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
+## PatternFly 5.2.1
+
+PatternFly 5.2.1 is now live! This was a patch release to fix the below bugs. Here are the package versions from this release.
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:12]:
```suggestion
PatternFly 5.2.1 is now live! This was a patch release to fix a few bugs that were found.
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 6]
```diff
@@ -7,6 +7,35 @@ section: get-started
...
+  - [@patternfly/react-table@5.2.1](https://www.npmjs.com/package/@patternfly/react-table/v/5.2.1)
+  - [@patternfly/react-tokens@5.2.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/5.2.1)
+
+### Fixes include:
+- Issues with styles and inetractability of cards in Chromium based browsers
+  - Additional details: https://github.com/orgs/patternfly/discussions/6381
+- Support for status modifiers on Modal titles
+- Type updates to react-charts to reflect changes in the Victory types
+- Reversion of a previous type change with our clipboard copy component that was unintentionally breaking
+- Pass through of more props to monaco-editor in our code editor component
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:35]:
```suggestion
### Fixed bugs
In this patch release, we:
- Corrected issues with styles and intractability of cards in Chromium-based browsers.
  - Additional details: https://github.com/orgs/patternfly/discussions/6381
- Added support for status modifiers on modal titles.
- Updated types used in react-charts to reflect changes in the Victory types.
- Reverted a previous type change within our clipboard copy component that was unintentionally breaking.
- Updated our code editor component to pass through more properties to monaco-editor. 
```

**Review by edonehoo — CHANGES_REQUESTED** [review]
Small edits mostly to adjust tense!

**Review by edonehoo — APPROVED** [review]

File: packages/documentation-framework/versions.json [hunk 7]
```diff
@@ -1,9 +1,33 @@
...
+        "@patternfly/react-charts": "7.2.2",
+        "@patternfly/react-code-editor": "5.2.1",
+        "@patternfly/react-core": "5.2.1",
+        "@patternfly/react-icons": "5.2.1",
+        "@patternfly/react-styles": "5.2.1",
+        "@patternfly/react-table": "5.2.1",
+        "@patternfly/react-drag-drop": "5.2.1",
+        "@patternfly/react-tokens": "5.2.1",
+        "@patternfly/react-catalog-view-extension": "5.0.0",
+        "@patternfly/react-component-groups": "5.0.0",
```
Comment (wise-king-sullyman) [inline packages/documentation-framework/versions.json:18]:
```suggestion
        "@patternfly/react-component-groups": "5.1.0",
```

**Review by edonehoo — APPROVED** [review]
Looks great!

**Pull request merged**